site/en/blog/chrome-86-deps-rems/index.md

```diff
@@ -13,14 +13,14 @@ description: >
 # How to add a new author
 # https://developer.chrome.com/docs/handbook/how-to/add-an-author/
 authors:
-  - josephmedley
+  - joemedley
 
 # Required
 date: 2020-09-03
 
 # Optional
 # Include an updated date when you update your post
-updated: 2020-09-03
+updated: 2022-07-04
```

Comment:
```suggestion
updated: 2020-09-03
```

Reply:
as per the handbook, please only update these dates if a significant change has been made, as it makes it look as if the content has been revised.